src/exercises/compile-and-run/README.md

```diff
@@ -28,3 +28,33 @@ The second test program is written in CHERI C:
 ```
 6. Run the binary: note how the length of the capability depends on the size of
    the type it points to.
+
+# Compile and run aarch64 and Morello programs
```

Comment:
Duplicating the text like this is sad, there's nothing architecture-specific here